bitcoin/src/blockdata/script/borrowed.rs

```diff
@@ -414,7 +414,7 @@ crate::internal_macros::define_extension_trait! {
                         // Note: We ensure the division happens at the end, since Core performs the division at the end.
                         //       This will make sure none of the implicit floor operations mess with the value.
 
-            crate::Amount::from_sat(sats)
+            crate::Amount::from_sat(sats).expect("FIXME: Can this be hit?")
```

Comment:
Since `dust_relay_fee` is variable, and is being multiplied by L404.. then yes, it can be hit is what I'd conclude.  That's why I had changed this to return an `Error`.

Reply:
Needs more thought, the idea of an out-of-range error for a function that returns dust relay fee seems a big odd.